Assets/Scripts/Events/Outcomes/AdventurersRemoved.cs

```diff
@@ -7,19 +7,16 @@ namespace Events.Outcomes
     [CreateAssetMenu(fileName = "Remove Adventurers Outcome", menuName = "Outcomes/Remove Adventurers")]
     public class AdventurersRemoved : Outcome
     {
-        public List<string> adventurerNames;
+        public int count;
         // To shreds, you say?
         public bool kill; // If they move to the graveyard or just disappear
         public override bool Execute()
         {
-            if (Manager.Adventurers.Available <= adventurerNames.Count) return false;
-        
-            foreach (string t in adventurerNames)
+            if (Manager.Adventurers.Available <= count) return false;
+
+            for (int i = 0; i < count; i++)
```

Comment:
Is this really the only C# syntax for ranges?

Reply:
You could use Enumerable.Range(0,count).select(...) but tbh this is the most convenient and readable